apps/src/music/player/MusicPlayer.js

```diff
@@ -256,7 +256,7 @@ export default class MusicPlayer {
   }
 
   convertSecondsToMeasure(seconds) {
```

Comment:
nit: can this be `convertSecondsToPlayheadTime` since it's returning an exact value (instead of a measure number)? Or we can even just move this calculation inline to `getCurrentPlayhead` since it's only used there, I believe

Reply:
Renamed this to `convertSecondsToPlayheadPosition`, and renamed the opposite function `convertPlayheadPositionToSeconds`.  It feels handy to have these both available, and as a bonus, they can do the conversion from 0-based to 1-based as well.